.github/workflows/release-hash.yml

```diff
@@ -2,25 +2,47 @@ name: Release Hash
 on:
   workflow_dispatch:
     inputs:
-      hash:
-        description: 'IPFS Hash (CID)'
-        required: true
-        default: ''
+      cid:
+        name: cid
+        description: IPFS Hash (if blank, pulled from build step)
+
+      signature:
+        name: Signature
+        description: Signature (if blank, pulled via Seacrest)
 jobs:
   build:
     name: Release Hash
     runs-on: ubuntu-latest
+    environment: Production
     steps:
-      - name: Checkout
+      - name: Checkout repository
         uses: actions/checkout@v2
 
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
-          node-version: '16.x'
+          node-version: '15.x'
```

Comment:
Does this not work on node 16?

Reply:
I was just copying this step from https://github.com/compound-finance/palisade/blob/main/.github/workflows/build-workflow.yml#L16-L19 -- I think we should actually bump up to Node 16 or 18 overall if we can.